### DISCUSSION TITLE
I started with Code OSS and I did not found the volar extension

### DISCUSSION THREAD

**Discussion by kasec** [vuejs/language-tools#84] [Q&A] [answered]
Recently I have installed a code oss version 1.53.2, I want to install volar vue extension, but my editor does not found, I'm looking if exist a way to install it.

**Comment by johnsoncodehk** [comment, marked as answer]
You also can click "Download Extension" from: https://marketplace.visualstudio.com/items?itemName=johnsoncodehk.volar

And than drop the `.vsix` file to VSCode to install.
